src/pages/404.page.jsx

```diff
@@ -1,11 +1,22 @@
-import Link from 'next/link';
 import Layout from '../components/Layout';
+import Header from '../components/Header';
 
 export default function ErrorPage() {
   return (
     <Layout pageTitle="404">
-      <h1>404 Not Found</h1>
-      <Link href="/">Overview Page</Link>
+      <section className="error-page-header">
+        <Header />
+      </section>
+      <div className="details-page__primary">
```

Comment:
I just have a general note that there's a _lot_ of spacing between everything. Not necessarily a bad thing, but when there's so little content on a page, it's weird to have to scroll to see all of it, you know?

Rather than reusing `details-page__primary`, I'd create a new component and work on styling that to be a little more compact.

![Screen Shot 2022-12-14 at 15 16 15](https://user-images.githubusercontent.com/82047862/207727291-a9e35f68-2702-4a95-bfce-085418768550.png)

Reply:
How's this instead? 
https://github.com/sparkbox/accessible-components/pull/136/commits/5e4e68eecde99ed9192668379941beee5088f797

<img width="742" alt="Screen Shot 2022-12-15 at 9 50 17 AM" src="https://user-images.githubusercontent.com/69602589/207892253-8090713d-f98e-4c54-ab05-9a1cdcf1fef8.png">